next.json

```diff
@@ -493,6 +547,42 @@
             }
         ]
     },
+    "render pass builder": {
+        "category": "object",
+        "methods": [
+            {
+                "name": "get result",
+                "returns": "render pass"
+            },
+            {
+                "name": "enable attachment",
+                "comment": "Lazy implementation; probably should make an Attachment object.",
+                "args": [
+                    {"name": "attachment slot", "type": "uint32_t"},
+                    {"name": "format", "type": "texture format"}
+                ]
+            },
+            {
+                "name": "set subpass count",
```

Comment:
Likewise we should probably have a "set attachment count" and validate that both are done before anything else?